From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Nicolas Iooss <nicolas.iooss_linux@m4x.org>
Date: Mon, 23 Apr 2018 21:19:16 +0200
Subject: [PATCH] {MAYBE UPS} objtool: silence -Wunreachable-code-return

clang reports:

    builtin-orc.c:67:9: error: 'return' will never be executed
    [-Werror,-Wunreachable-code-return]
            return 0;
                   ^
---
 tools/objtool/builtin-orc.c | 2 --
 tools/objtool/check.c       | 2 --
 2 files changed, 4 deletions(-)

diff --git a/tools/objtool/builtin-orc.c b/tools/objtool/builtin-orc.c
index 8273bbf7cebb..6047c70fbae5 100644
--- a/tools/objtool/builtin-orc.c
+++ b/tools/objtool/builtin-orc.c
@@ -71,6 +71,4 @@ int cmd_orc(int argc, const char **argv)
 	}
 
 	usage_with_options(orc_usage, check_options);
-
-	return 0;
 }
diff --git a/tools/objtool/check.c b/tools/objtool/check.c
index 5e5388a38e2a..2630bb6ca5c6 100644
--- a/tools/objtool/check.c
+++ b/tools/objtool/check.c
@@ -2853,8 +2853,6 @@ static int validate_branch(struct objtool_file *file, struct symbol *func,
 
 		insn = next_insn;
 	}
-
-	return 0;
 }
 
 static int validate_unwind_hints(struct objtool_file *file, struct section *sec)
-- 
